### ISSUE TITLE
Blank map when center parameter used in form input

### ISSUE THREAD

**Issue by JeroenDeDauw** [SemanticMediaWiki/SemanticMaps#8]
From http://sourceforge.net/p/semediawiki/mailman/message/32131234/
## 

I discovered this when trying to use a map for entering coordinates into a
form.  I copied what had been working previously and found that what
displayed was basically a blank map.  I narrowed down the problem to the
center parameter.  If I drop that, the map will default as centered at
coordiantes 0,0.  I don't think this problem is related to the SemanticMaps
not showing up, since it also doesn't work on another wiki that has both
Maps and SemanticMaps 3.0 showing on the Special:Version page.

In summary:
'''location Coordinates:''' {{{
 field
| Coordinates
| input type=googlemaps
| type=satellite
| height=500
| width=500
| autozoom=off
| zoom=8
}}}

works (but starts showing a portion of the Atlantic Ocean off of Africa),
but
'''location Coordinates:''' {{{
 field
| Coordinates
| input type=googlemaps
|center=57,-135
| type=satellite
| height=500
| width=500
| autozoom=off
| zoom=8
}}}

does not work, showing a blank map with controls that don't seem to do
anything.
